corehq/apps/repeaters/models.py

```diff
@@ -132,6 +68,8 @@ class Repeater(QuickCachedDocumentMixin, Document, UnicodeMixIn):
     password = StringProperty()
     friendly_name = _("Data")
 
+    payload_generator_classes = ()
```

Comment:
@proteusvacuum OK I think I finally nailed it. As @NoahCarnahan suggested, it'd ideally just be a property—and I then realized/remembered that `tuple`s don't trigger the same overzealous response in a `couchdbkit.Document`, so just using `tuple`s is an easy work around I think.